libnl/nl80211/iw_scan.py

```diff
@@ -504,7 +504,14 @@ def get_secchan_offs(type_, data):
     type_ -- corresponding `ieprinters` dictionary key for the instance.
     data -- bytearray data to read.
     """
-    raise NotImplementedError
+    chans = [
+        "no secondary",
+        "above",
+        "[reserved!]",
+        "below"]
+    if data[0] < len(chans):
+        return chans[data[0]]
```

Comment:
Looking at the upstream C source this function should return `"{0} ({1})".format(ht_secondary_offset[data[0]], data[0])` for this condition.